Sources/OpenTracing/Format.swift

```diff
@@ -3,24 +3,47 @@ import Foundation
 /// Format and carrier for extract(). Marker protocol.
 public protocol FormatReader {}
 
-// Format and carrier for inject(). Marker protocol.
+/// Format and carrier for inject(). Marker protocol.
 public protocol FormatWriter {}
 
+/// Read interface for a textmap
 public protocol TextMapReader: FormatReader {
+
+    /// Return the text map as a [String: String]
```

Comment:
Maybe explain what this contains instead of the type which is inferred? (i.e., what's a text map?)

Reply:
Questions like "what's a text map" don't belong in header documentation as they are OpenTracing concepts. I will however add links to relevant OpenTracing documentation explaining these concepts at the top of the file.